spec/features/staff/beis_users_can_edit_a_user_spec.rb

```diff
@@ -43,6 +43,24 @@
     expect(page).to have_content("New Name")
   end
 
+  scenario "the user's MFA (mobile number and confirmed_at) can be reset" do
+    administrator_user = create(:beis_user)
+    authenticate!(user: administrator_user)
+
+    # Navigate to the users page
+    visit users_path
+
+    find("tr", text: user.name).click_link("Edit")
+
+    # When I reset the user's MFA
+    check "Reset the mobile number used for authentication"
+    click_on t("default.button.submit")
+
+    # Then that user should no longer be confirmed for MFA
+    expect(page).to have_content("User successfully updated")
+    expect(page).to have_content("No")
```

Comment:
Matching a string as short as 'No' feels very error-prone; e.g. this would pass if the user was deactivated but had confirmed their phone number.

I haven't played with `have_text` so don't know if the output is nice, but you might be able to say something like `expect(page).to have_text("Mobile number confirmed for authentication? No")` or similar.

Reply:
Thank you, `have_content` did the job!